index.html

```diff
@@ -26,19 +26,18 @@
   <meta ng-bind="title" content="Standard Notes" name="application-name" />
 
   <title>Dev · Notes · Standard Notes</title>
+</head>
 
+<body
+  data-default-sync-server="<%= env.DEV_DEFAULT_SYNC_SERVER %>"
+  data-extensions-manager-location="<%= env.DEV_EXTENSIONS_MANAGER_LOCATION %>"
+  data-batch-manager-location="<%= env.DEV_BATCH_MANAGER_LOCATION %>"
+>
   <script>
-    window._default_sync_server = "https://sync.standardnotes.org";
-    window._extensions_manager_location = "public/extensions/extensions-manager/dist/index.html";
-    window._batch_manager_location = "public/extensions/batch-manager/dist/index.min.html";
+    window._default_sync_server = document.body.dataset.defaultSyncServer || "https://sync.standardnotes.org";
```

Comment:
This should not be used anymore actually.

Reply:
Thanks, I've added a commit to set this value in SNJS directly. We still need it on web because this can also be set up by rails at runtime (i-e we can't set the value during build time because it would cripple the docker image)